docs/api-docs/crossplane.md

```diff
@@ -1,7 +1,5 @@
 ---
-title: crossplane
-toc: true
-weight: 401
-indent: true
-redirect_to: https://doc.crds.dev/github.com/crossplane/crossplane
----
+title: Crossplane
+layout: redirect
+to: "https://crossplane.io/docs/v1.9/api-docs/crossplane.html"
```

Comment:
Is this a vanity link? If so, should it use `master` on `master` branch?

Reply:
This was an archive from testing, good catch.